### ISSUE TITLE
make install fails with 'Unable to communicate with docker daemon.'

### ISSUE THREAD

**Issue by martaver** [cloudposse/root.cloudposse.co#6]
Running with on WSL with Ubuntu 16.04

`make init`, `make docker/build` commands works fine. build-harness is installed and Dockerfile is built. Clearly, make can operate docker fine.

When I run `make install` I get the following error:
```
Unable to communicate with docker daemon. Make sure your environment is properly configured and then try again.
Makefile:29: recipe for target 'install' failed
make: *** [install] Error 1
```

Any clues?

**Comment by Martaver** [comment]
The problem was that in WSL, I'm mounting windows' docker daemon, and docker needs the DOCKER_HOST environment variable to point it to the right host. In the 'install' target for make, docker is run in sudo, which doesn't automatically import all env vars. By adding the `-E` argument, it allows sudo to preserve all the vars and gives docker what it needs. 

Updated install target like so:
```
## Install wrapper script from geodesic container
install:
	@docker run --rm $(DOCKER_IMAGE_NAME) | sudo -E bash -s $(DOCKER_TAG)
```

**Comment by osterman** [comment]
Wait, you’re using windows?! Wow, we’ve never tested that. I want to document how to achieve that.

**Comment by osterman** [comment]
@vadim-hleif can you give this a go and see if it fixes your problem as well?

**Comment by Martaver** [comment]
Its WSL, not powershell, mind.

**Comment by osterman** [comment]
Cool, yea - @vadim-hleif is a new guy on my team and recently started trying to get this working in the [Ubuntu shell for Windows](https://docs.microsoft.com/en-us/windows/wsl/about). I'm going to have him pick up from where you left off since he has a windows machine.

**Comment by vadim-hleif** [comment]
Fixed in [#197](https://github.com/cloudposse/geodesic/pull/197)
